packages/ERTP/test/swingsetTests/splitPayments/bootstrap.js

```diff
@@ -19,7 +21,9 @@ export function buildRootObject(vatPowers, vatParameters) {
           return testSplitPayments(aliceMaker);
         }
         default: {
-          throw Error(`unrecognized argument value ${vatParameters.argv[0]}`);
+          assert.fail(
+            d`unrecognized argument value ${vatParameters.argv[0]}`,
```

Comment:
I would make `vatParameters.argv[0]` a `const` so that this could just refer to it. That general pattern protects against TOCTOU and makes this fit on one line.